packages/react-components/src/components/VideoGallery/RemoteScreenShare.tsx

```diff
@@ -60,7 +60,7 @@ export const RemoteScreenShare = React.memo(
 const LoadingSpinner = (props: { loadingMessage: string }): JSX.Element => {
   return (
     <Stack verticalAlign="center" className={loadingStyle}>
-      <Spinner label={props.loadingMessage} size={SpinnerSize.xSmall} />
+      <Spinner label={props.loadingMessage} size={SpinnerSize.xSmall} aria-live={'assertive'} />
```

Comment:
is there a constant we can use here? I can imagine someone could just miss spell this

Reply:
we can create one, sure

Reply:
I would actually argue against creating a constant here as aria-live has a specific interface: 
![image](https://user-images.githubusercontent.com/2684369/144891107-e9b31c29-f1fe-49cf-a11e-36583c72410c.png)

Reply:
looking at that screenshot looks like we need to use ariaLive and not aria-live to get that type checking though

Reply:
will do in another PR though

Reply:
@JamesBurnside 'ariaLive' prop from Spinner FluientUI component is not working properly when inside a Stack, that's why I'm using the 'aria-live' from AriaAttributes to force the behaviour of Spinner

Reply:
> 

the 'ariaLive' prop of Spinner component has indeed a specific interface, but not the 'aria-live' attribute

Reply:
but, yes, you're right, even the AriaAttributes have specific interface, so no need for constants here then